src/components/Table/SubRows.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { Table, SubRowTableRow } from './styles';
+
+type Props = {
+  colSpan: number;
+  children: React.ReactNode | React.ReactNode[];
+  gutter: number | { left: number; bottom: number };
+};
+
+const SubRows: React.FC<Props> = ({ colSpan, gutter, children, ...rest }: Props) => {
+  const left = typeof gutter === 'number' ? gutter : gutter.left;
+  const bottom = typeof gutter === 'number' ? gutter : gutter.bottom;
+
+  return (
+    <SubRowTableRow isStatic {...rest}>
+      <td colSpan={colSpan} style={{ paddingLeft: left, paddingBottom: bottom }}>
+        <Table>{children}</Table>
```

Comment:
Didn't this give an error ? since you can pass a DIV in children.
( so you miss the TR tag )

Reply:
Yep, you're right, forgot about that, I'll add the TableBody component.